Automattic/jetpack
# Search: Clicking outside overlay closes overlay

Fixes #18937.

#### Changes proposed in this Pull Request:
Adds an event handler on the window checking for clicks outside the search overlay.

#### Does this pull request change what data or activity we track or use?
None.

#### Testing instructions:






* Apply this change to a site with a Jetpack Search subscription and Instant Search enabled.
* Perform a site search `e.g. `/?s=page`.
* Click outside the overlay.
* Ensure that the overlay closes as expected.
